src/profile/v100/entity/display_name.rs

```diff
@@ -1,5 +1,7 @@
 use crate::prelude::*;
 
+json_data_convertible!(DisplayName);
```

Comment:
hmm this should be `json_string_convertible`

Reply:
@CyonAlexRDX  Nice! I should use `json_string_convertible` on `radix_connect_purpose.rs` too, right?

Reply:
Yes it makes sense for that type too